### DISCUSSION TITLE
where to get the wallpaper resourse?

### DISCUSSION THREAD

**Discussion by rabbit3046** [rocksdanister/lively#1651] [Q&A]
where to get the wallpaper resourse?

**Comment by rocksdanister** [comment]
Do you mean wallpapers?

Gallery is not released, once ready can be downloaded from app.

Some wallpapers can be found here:

https://github.com/rocksdanister/audio-visualizer-wallpaper

https://github.com/rocksdanister/system-stats-wallpaper

Some wallpapers are uploaded to deviantart group:

https://www.reddit.com/r/LivelyWallpaper/comments/kfvc2o/how_to_download_wallpapers_from_lively_deviantart/
